Framework/AODMerger/src/aodThinner.cxx

```diff
@@ -367,15 +384,6 @@ int main(int argc, char* argv[])
           }
         }
 
-        // Reassign v0 index of cascades
-        if (processingCascades) {
-          if (keepV0s[indexV0s] < 0) {
-            fillThisEntry = false;
-          } else {
-            indexV0s = keepV0s[indexV0s];
-          }
-        }
-
```

Comment:
So this part is gone because we keep all V0s and cascades, right?

Reply:
yes